.github/workflows/build.yml

```diff
@@ -1,13 +1,11 @@
+# Build the book and upload it to GitHub Artifacts for inspection in a PR
 name: build
 
 # Run on every pull request
 on:
-  push:
```

Comment:
removing this because I think it is redundant with `deploy.yml` if we push to master